indra/llcharacter/lljoint.cpp

```diff
@@ -209,8 +210,10 @@ void LLJoint::touch(U32 flags)
 			child_flags |= POSITION_DIRTY;
 		}
 
-		for (LLJoint* joint : mChildren)
+		for (joints_t::iterator iter = mChildren.begin();
+			 iter != mChildren.end(); ++iter)
 		{
+			LLJoint* joint = *iter;
```

Comment:
Revert the above change.  `mChildren` really is a container of raw pointers and the old pattern is preferred.